docs_outdoornav_user_manual/customized_tuning/tuning_instructions/differential_drive_dynamics.mdx

```diff
@@ -21,28 +28,33 @@ flowchart. Begin with setting up [Initial Parameters](#initialize_parameters) fo
   </figure>
 </center>
 
-## Conditions Blocks {#condition_blocks}
+**Condition Blocks**: The condition blocks can be seen in the flowchart as **RED** decision 
+blocks and either contain tests needed to be run or features of the software that are 
+enabled/disabled.
 
-The condition blocks can be seen in the flowchart as **RED** decision blocks and either contain tests needed to be run or features of the software that are enabled/disabled.
+**Tuning Blocks**: The tuning blocks can be seen in the flowchart as **YELLOW** blocks where it
+is expected that the user modifies a parameter prior to re-testing the controller.
 
+## Conditions Blocks {#condition_blocks}
 
 #### Velocity Deadband Tests {#velocity_deadband_tests}
 
-The "**Velocity Deadband?**" condition block asks the user to verify if there is a velocity deadband that the platforms
-motors must overcome in order to move the platform. (ie. what is the minimum velocity that the platform
+The "**Velocity Deadband?**" condition block asks the user to verify if there is a velocity deadband that the UGV
+motors must overcome in order to move the UGV. (ie. what is the minimum velocity that the UGV
 is required to receive, in order for it to move).
 
 To verify this, the user should set up two missions:
 
-1. A single Waypoint that is 10-20 meters directly in front of the platform. This will verify proper linear motion
-of the platform.
-2. A single waypoint that is 10-20 meters directly behind the platform. The first part of the navigation to
-this point (ie. the rotation to face the Waypoint) will verify the angular motion of the platform.
+1. A single Waypoint that is 10-20 meters directly in front of the UGV. This will verify 
```

Comment:
Is this still valid now? If the UGV needs to be withing 1-2 meters to start a mission will this still work?

Reply:
yes good catch. I will fix this.